tests/component/_task_modules/test_triggers_properties.py

```diff
@@ -99,3 +109,43 @@ def test___ai_task___reset_uint32_property___returns_default_value(ai_voltage_ta
     del ai_voltage_task.triggers.reference_trigger.pretrig_samples
 
     assert ai_voltage_task.triggers.reference_trigger.pretrig_samples == 2
+
+
+def test___ai_voltage_field_daq_task___get_timestamp_property___returns_default_value(
+    ai_voltage_field_daq_task: Task,
+):
+    ai_voltage_field_daq_task.timing.cfg_samp_clk_timing(1000)
+    when_value = ai_voltage_field_daq_task.triggers.start_trigger.trig_when
+    assert when_value.lsb == 0
+    assert when_value.msb == 0
+
+
+def test___ai_voltage_field_daq_task___set_timestamp_property___returns_assigned_value(
+    ai_voltage_field_daq_task: Task,
+):
+    value_to_test = JAN_01_2002_HIGHTIME
+
+    ai_voltage_field_daq_task.timing.cfg_samp_clk_timing(1000)
+    ai_voltage_field_daq_task.triggers.start_trigger.trig_when = value_to_test
+    when_value = ai_voltage_field_daq_task.triggers.start_trigger.trig_when
+
+    when_value_dt = when_value.to_datetime(timezone.utc)
+    assert when_value_dt.year == 2002
+    assert when_value_dt.month == 1
+    assert when_value_dt.day == 1
+    assert when_value_dt.hour == 0
+    assert when_value_dt.minute == 0
+    assert when_value_dt.second == 0
+
+
+def test___ai_voltage_field_daq_task___reset_timestamp_property___returns_default_value(
+    ai_voltage_field_daq_task: Task,
+):
+    ai_voltage_field_daq_task.timing.cfg_samp_clk_timing(1000)
+    ai_voltage_field_daq_task.triggers.start_trigger.trig_when = JAN_01_2002_HIGHTIME
+
+    del ai_voltage_field_daq_task.triggers.start_trigger.trig_when
+    when_value = ai_voltage_field_daq_task.triggers.start_trigger.trig_when
+
+    assert when_value.lsb == 0
+    assert when_value.msb == 0
```

Comment:
arrange, act, assert
```suggestion
    ai_voltage_field_daq_task.timing.cfg_samp_clk_timing(1000)
    ai_voltage_field_daq_task.triggers.start_trigger.trig_when = JAN_01_2002_HIGHTIME

    del ai_voltage_field_daq_task.triggers.start_trigger.trig_when
    
    when_value = ai_voltage_field_daq_task.triggers.start_trigger.trig_when
    assert when_value.lsb == 0
    assert when_value.msb == 0
```